src/Microsoft.Health.Fhir.Core/Data/OperationDefinition/group-export.json

```diff
@@ -0,0 +1,45 @@
+{
+    "resourceType": "OperationDefinition",
+    "id": "group-export",
+    "url": "[base]/OperationDefinition/group-export",
+    "version": "1.0.0",
+    "name": "Group Export",
+    "title": "FHIR Bulk Data Export - Group Export",
+    "status": "active",
+    "kind": "operation",
+    "description": "Export all patient compartment data that are part of the specified group. More details regarding bulk export can be found [here] [http://hl7.org/fhir/uv/bulkdata/export/index.html]. This operation is asynchronous as defined in the [FHIR Asynchronous Request Pattern](http://hl7.org/fhir/async.html)",
+    "code": "export",
+    "resource" : [
+        "Group"
+      ],
+    "base": "/",
+    "system": false,
+    "type": true,
+    "instance": false,
```

Comment:
Group is an instance level operation, not a type level.

Reply:
Nice catch! Updated